src/v/cluster/tx_gateway_frontend.cc

```diff
@@ -1160,6 +1161,55 @@ ss::future<end_tx_reply> tx_gateway_frontend::end_txn(
       });
 }
 
+ss::future<tx_gateway_frontend::return_all_txs_res>
+tx_gateway_frontend::get_all_transactions() {
+    auto shard = _shard_table.local().shard_for(model::tx_manager_ntp);
+
+    if (!shard.has_value()) {
+        vlog(txlog.warn, "can't find a shard for {}", model::tx_manager_ntp);
+        co_return tx_errc::shard_not_found;
+    }
+
+    co_return co_await container().invoke_on(
+      *shard,
+      _ssg,
+      [](tx_gateway_frontend& self)
+        -> ss::future<tx_gateway_frontend::return_all_txs_res> {
+          auto partition = self._partition_manager.local().get(
+            model::tx_manager_ntp);
+          if (!partition) {
+              vlog(
+                txlog.error,
+                "can't get partition by {} ntp",
+                model::tx_manager_ntp);
+              co_return tx_errc::partition_not_found;
+          }
```

Comment:
I'm still concerned about this being logged at error level. If I understand correctly this condition is triggered if an RPC arrives at a node and the partition has moved. Presumably the client will retry. The concern is that this is a normal (but infrequently occurring case) but since it is logged at an error level log monitoring systems will raise an alert. @rystsov any thoughts here on error level?

Reply:
Oh, I did not think in this direction (log monitoring system).

Reply:
I agree error level is excessive since it's more or less expected situation, lets change it to warn